staking/integration-tests/tests/delegate.rs

```diff
@@ -278,24 +264,6 @@ fn test_delegate() {
         assert_eq!(pool_data.self_del_state[i], DelegationState::default());
     }
 
-    svm.expire_blockhash();
-    advance_delegation_record(
```

Comment:
removing this allows us to check that the undelegate right after works without advancing delegation record